### ISSUE TITLE
Certain links do not open in Firefox when logging in without running startx

### ISSUE THREAD

**Issue by mihaimorariu** [fairyglade/ly#339]
# Environment

OS: Arch Linux 5.16.9-arch1-1
Browser: Firefox 97.0.1
Window manager: i3 version 4.20.1
Display manager: ly 0.5.3-0

# Description

If I select "i3" at the log in screen, clicking links in certain apps (e.g., Slack, Signal) doesn't do anything. If I select "shell" instead of "i3", then type "startx", clicking links in the same app(s) works.

This problem has been reported by other users as well (https://www.reddit.com/r/archlinux/comments/oua48x/electron_links_not_opening_in_browser/) and switching to a different display manager or to Chrome seems to have fixed the issue. I really like Firefox and ly and would like to continue to use both of them without having to log in via the shell.

Any ideas on how to fix this? 🙏🏻

**Comment by mihaimorariu** [comment]
Thanks for your reply!

I have tested with the latest changes, but, unfortunately, the problem still persists. I also tried lxdm (as some Reddit users have suggested) and no longer faced this problem.

**Comment by CrookedDuck** [comment]
i also struggled a lot with this, but strangely before switching to ly :) 
have you tried the suggestions from:

https://www.reddit.com/r/archlinux/comments/oua48x/electron_links_not_opening_in_browser/
https://github.com/electron/electron/issues/28436#issuecomment-838583487
https://bbs.archlinux.org/viewtopic.php?id=257535

tl;dr: try 
`export GDK_BACKEND=x11`
or
`export MOZ_ENABLE_WAYLAND=0`
